package.json

```diff
@@ -35,25 +35,26 @@
     "update-dependencies": "npx npm-check-updates --upgrade --dep prod,dev,optional,bundle"
   },
   "dependencies": {
-    "@cucumber/messages": "^13.2.1",
     "ansi-styles": "^5.0.0",
     "cli-table3": "^0.6.0",
     "figures": "^3.2.0",
     "ts-dedent": "^2.0.0"
   },
   "peerDependencies": {
-    "@cucumber/cucumber": ">=7.0.0"
+    "@cucumber/cucumber": ">=7.0.0",
+    "@cucumber/messages": "*"
```

Comment:
`messages` should be fine as a peer and dev dependency - we know `cucumber` will always bring it along, and it's only needed here for TypeScript reasons - once we compile down to JS there is no import of `messages`.